portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includePage.vm

```diff
@@ -115,27 +115,27 @@
 
                                 #end ## END of IF ( $toolDirectUrlEnabled && ${tool.showDirectToolUrl} )
 
-                                #if (${tool.toolShowHelpButton})
+                        <script>
+                             // if the role switcher is present the page title 'overview' then add padding-right to the add content button
 
-                                    #if (${tool.toolJSR168Help})
+                              const roleSwitcher = document.getElementById("roleSwitch");
+                              const addContentButton = document.querySelector(".Mrphs-toolTitleNav__button_container");
 
-                                        <a class="Mrphs-toolTitleNav__link Mrphs-toolTitleNav__link--help" accesskey="h" href="${tool.toolJSR168Help}" title="$rloader.getFormattedMessage("sit_helpfor", $tool.toolRenderResult.title)">
-                                            <span class="Mrphs-itemTitle">${rloader.sit_help}</span>
-                                            <span class="skip">${rloader.site_newwindow}</span>
-                                        </a>
-
-                                    #else
-
-                                        <a class="Mrphs-toolTitleNav__link Mrphs-toolTitleNav__link--help-popup" href="${tool.toolHelpActionUrl}" title="$rloader.getFormattedMessage("sit_helpfor", $tool.toolRenderResult.title)" target="_blank" onclick="openWindow('${tool.toolHelpActionUrl}', 'Help', 'resizable=yes,toolbar=no,scrollbars=yes,menubar=yes,width=1024,height=768'); return false">
-                                            <span class="Mrphs-itemTitle">${rloader.sit_help}</span>
-                                            <span class="skip">${rloader.site_newwindow}</span>
-                                        </a>
+                              if (roleSwitcher && addContentButton) {
 
-                                    #end ## END of IF (${tool.toolJSR168Help})
+                                    addContentButton.style.paddingRight = "10.5em";
```

Comment:
These lines on CSS file?

Reply:
I don't think I can put that inside the CSS file because it only needs to be applied when roleSwitcher is present on the page